src/tinyformat.h

```diff
@@ -1061,6 +1061,24 @@ std::string format(const std::string &fmt, const Args&... args)
     return oss.str();
 }
 
+/** Variant of tfm::format that aborts when an formatting exception happens instead
+ * of raising an exception.
+ *
+ * Use this function in no-except context, when you are 100% sure that the
+ * format string contains no errors. Only errors in the format string and inconsistencies
+ * between the format string and the type or number of arguments passed in cause tinyformat
+ * exceptions. It does not depend on the value of the arguments.
+ */
+template<typename... Args>
+std::string format_noexcept(const std::string &fmt, const Args&... args) noexcept
```

Comment:
Could make this shorter by removing the copy-pasted body?

```suggestion
std::string format_noexcept(const std::string &fmt, const Args&... args) noexcept {return format(fmt, args...);}
```

Reply:
it looks really weird like this, having ` _noexcept` that simply calls the normal function

Reply:
But it is exactly what happens here. Inlining `format` doesn't make it throw less exceptions. The only difference between them is the `noexcept` keyword, which makes the program terminate immediately should any exception be thrown.

Reply:
yes, that's true

Reply:
OTOH this is only three lines, I'd like to keep the comment too, and this adds yet another layer of indirection with `std::string` return value